frontend/src/component/common/AvatarGroup/AvatarGroup.tsx

```diff
@@ -34,13 +38,17 @@ type AvatarGroupProps = {
     avatarLimit?: number;
     AvatarComponent?: typeof UserAvatar;
     className?: string;
+    margin?: number;
```

Comment:
I'm wondering if having those special case fields is a good way to go. Usually we compose new styling on top of the lower level elements. Could we make this component styleable with styled components/sx/styled(Avatar)?